pyproject.toml

```
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "hist-smooth"
+version = "0"

```

```suggestion
version = "0.1.0"
```

Let's use semantic versioning (SemVer) for this library.

---

Resolved